CHANGELOG.md

```diff
@@ -12,15 +12,17 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 -   New component in React: `FlexBox`
 -   New component in React: `Mosaic`
 -   Added `size` prop to `WebBookmark` component
+-   Added `badge` prop to `Avatar` component
+-   Change `avatar` prop of `UserBlock` component from `string` to `AvatarProp` to pass badge
```

Comment:
It's a breaking change. And actually maybe we could have the `avatar` prop be `string | AvatarProps` so that's retro-compatible?

Reply:
Done https://github.com/lumapps/design-system/pull/355/commits/4f4ff585e92dea90548ef2d92b73f00f5c56a4f4